modules/videopress/utility-functions.php

```diff
@@ -198,3 +199,59 @@ public function import( $args ) {
 	}
 	WP_CLI::add_command( 'videopress', 'VideoPress_CLI' );
 }
+
+/**
+ * Return an absolute URI for a given filename and guid on the CDN.
+ * No check is performed to ensure the guid exists or the file is present. Simple centralized string builder.
+ *
+ * @param string $guid     VideoPress identifier
+ * @param string $filename name of file associated with the guid (video file name or thumbnail file name)
+ *
+ * @return string Absolute URL of VideoPress file for the given guid.
+ */
+function videopress_cdn_file_url( $guid, $filename ) {
+	if ( is_ssl() ) {
+		return "https://videos.files.wordpress.com/{$guid}/{$filename}";
+
+	} else {
```

Comment:
Is there any benefit to serving this over http instead of https?

Also, I'd sooner drop the `else` and just have it be like ..

```php
if ( thing ) {
    return 'TRUE!';
}
return 'FALSE :(';
```

It just comes a touch more readable without the logical else, to my mind, at least.